src/content/docs/en/reference/errors/duplicate-content-entry-slug-error.mdx

```diff
@@ -0,0 +1,24 @@
+---
+# NOTE: This file is auto-generated from 'scripts/error-docgen.mjs'
+# Do not make edits to it directly, they will be overwritten.
+# Instead, change this file: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+# Translators, please remove this note and the <DontEditWarning/> component.
+
+title: Duplicate content entry slug.
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+import DontEditWarning from '~/components/DontEditWarning.astro'
+
+<DontEditWarning />
+
+
+> **DuplicateContentEntrySlugError**: 
+            return `**COLLECTION** contains multiple entries with the same slug: `SLUG`. Slugs must be unique.`;
```

Comment:
Blocking because this message seems malformed.

Same problem here: https://docs.astro.build/en/reference/errors/locals-not-serializable/

(Which doesn’t seem to be in the main error index page for some reason.)

Reply:
Looks like that Locals Not Serializable error isn’t in the index because it doesn’t exist in Astro any more — the error docgen adds errors, but won’t remove deleted errors. I think that’s OK because someone on an old Astro version might still hit the error and try visiting the link, so keeping it around is harmless — we can fix in docs instead of in Astro.